cypress/testcases/dashboard/support.spec.ts

```diff
@@ -31,9 +31,8 @@ describe("Support Page", () => {
       const kubeconfig = `${Cypress.config("downloadsFolder")}/local.yaml`
       page.downloadKubeConfigBtn.click()
 
-      // cy.readFile(kubeconfig)
-      //   .should("exist")
-      cy.verifyDownload('local.yaml', {timeout: constants.timeout.downloadTimeout});
```

Comment:
I tend to use `.readFile`, it will print file's content to log. (so we can track it if need)